src/main/webapp/resources/js/pages/projects/ncbi/create/CreateNcbiExport.tsx

```diff
@@ -0,0 +1,443 @@
+        bioProject: string;
+        namespace: string;
+        organization: string;
+        releaseDate: moment.Moment;
+        samples: {
+          string: {
+            files: {
+              pairs?: number[];
+              singles?: number[];
+            };
+            bioSample: string;
+            libraryName: string;
+            libraryStrategy: string;
+            librarySource: string;
+            libraryConstructionProtocol: string;
+            instrumentModel: [string, string];
+            librarySelection: string;
+          };
+        };
+      }) => {
+        setCreateStatus(CreateStatus.PENDING);
+        const request: NcbiSubmissionRequest = {
+          projectId: Number(projectId),
+          bioProject,
+          namespace,
+          organization,
+          releaseDate: releaseDate.unix(),
+          samples: Object.values(_samples).map(
+            ({
+              files = { pairs: [], singles: [] },
+              instrumentModel,
+              ...rest
+            }): NcbiSubmissionBioSample => {
+              return {
+                ...rest,
+                instrumentModel: instrumentModel[1],
+                singles: files.singles ? files.singles : [],
+                pairs: files.pairs ? files.pairs : [],
+              };
+            }
+          ),
+        };
+
+        submitNcbiSubmissionRequest(request)
+          .then(() => {
+            setCreateStatus(CreateStatus.RESOLVED);
+            // Redirect the user back to the referrer page
+            setTimeout(() => navigate(-1), 2000);
+          })
+          .catch((message = i18n("CreateNcbiExport.error")) => {
+            setCreateStatus(CreateStatus.REJECTED);
+            notification.error({
+              message,
+            });
+          });
+      }
+    );
+  };
+
+  /**
+   * Remove a sample from the export.
+   * Note: this only removes from the export, but if the page is reloaded, they still
+   * exist in the sessionStorage.
+   * @param event
+   * @param sample - sample to remove
+   */
+  const removeSample = (
+    event: React.MouseEvent<HTMLElement>,
+    sample: SampleRecord
+  ): void => {
+    event.stopPropagation();
+    const updated = { ...samples };
+    delete updated[sample.name];
+    setSamples(updated);
+  };
+
+  const buttons =
+    createStatus !== CreateStatus.RESOLVED ? (
+      <Button
+        type="primary"
+        htmlType="submit"
+        loading={createStatus === CreateStatus.PENDING}
+        disabled={createStatus === CreateStatus.PENDING}
+        className="t-submit-button"
+      >
+        {i18n("CreateNcbiExport.submit")}
+      </Button>
+    ) : (
+      <Alert type="success" message={i18n("CreateNcbiExport.success")} />
+    );
+
+  return (
+    <Layout.Content>
+      <Row justify="center">
+        <Col xxl={16} xl={20} sm={24}>
+          <PageHeader title={i18n("CreateNcbiExport.title")}>
+            {Object.keys(samples).length > 0 ? (
+              <>
```

Comment:
Is this empty fragment required?

Reply:
Looks like your right, removed in 24bc1d198d917bb42890e217cd32a014cc04ac54